main
NOT RUN! Job state is: New      
JobID: [any://localhost]-[rendvous-fn1:22578]
Just RUN! Job state is: Running  
SUSPENDED! Job state is: Suspended
RESUMED! Job state is: Running  
NOT DONE! Job state is: Running  
AWAITED! Job state is: Running  
should be DONE by now! Job state is: Running  
JobID: [any://localhost]-[rendvous-fn1:22583]
Just RUN! Job state is: Running  
CANCELED! Job state is: Canceled 
kortas@rendvous-fn1:~/XTREEMOS/TESTS/SAGA/EX$ main
NOT RUN! Job state is: New      
JobID: [any://localhost]-[rendvous-fn1:22586]
Just RUN! Job state is: Running  
terminate called after throwing an instance of 'saga::exception'
  what():  SAGA(NoSuccess): run_mode_wrapper.hpp(83): 
  SAGA(NoSuccess): default_job: process_helpers.hpp(226): Couldn't suspend job: No such process

Abandon
kortas@rendvous-fn1:~/XTREEMOS/TESTS/SAGA/EX$ main
NOT RUN! Job state is: New      
JobID: [any://localhost]-[rendvous-fn1:22589]
Just RUN! Job state is: Running  
terminate called after throwing an instance of 'saga::exception'
  what():  SAGA(NoSuccess): run_mode_wrapper.hpp(83): 
  SAGA(NoSuccess): default_job: process_helpers.hpp(226): Couldn't suspend job: No such process

Abandon
kortas@rendvous-fn1:~/XTREEMOS/TESTS/SAGA/EX$ main
NOT RUN! Job state is: New      
JobID: [any://localhost]-[rendvous-fn1:22592]
Just RUN! Job state is: Running  
terminate called after throwing an instance of 'saga::exception'
  what():  SAGA(NoSuccess): run_mode_wrapper.hpp(83): 
  SAGA(NoSuccess): default_job: process_helpers.hpp(226): Couldn't suspend job: No such process

Abandon
kortas@rendvous-fn1:~/XTREEMOS/TESTS/SAGA/EX$ cvs

